lib/auth/grpcserver.go

```diff
@@ -2443,6 +2447,35 @@ func (g *GRPCServer) DeleteAllNodes(ctx context.Context, req *types.ResourcesInN
 	return &empty.Empty{}, nil
 }
 
+// GetClusterAuditConfig gets cluster audit configuration.
+func (g *GRPCServer) GetClusterAuditConfig(ctx context.Context, _ *empty.Empty) (*types.ClusterAuditConfigV2, error) {
+	auth, err := g.authenticate(ctx)
+	if err != nil {
+		return nil, trail.ToGRPC(err)
+	}
+	auditConfig, err := auth.ServerWithRoles.GetClusterAuditConfig(ctx)
+	if err != nil {
+		return nil, trail.ToGRPC(err)
+	}
+	auditConfigV2, ok := auditConfig.(*types.ClusterAuditConfigV2)
+	if !ok {
+		return nil, trail.ToGRPC(trace.BadParameter("unexpected type %T", auditConfig))
+	}
+	return auditConfigV2, nil
+}
+
+// SetClusterAuditConfig sets cluster audit configuration.
+func (g *GRPCServer) SetClusterAuditConfig(ctx context.Context, auditConfig *types.ClusterAuditConfigV2) (*empty.Empty, error) {
+	auth, err := g.authenticate(ctx)
+	if err != nil {
+		return nil, trail.ToGRPC(err)
+	}
+	if err = auth.ServerWithRoles.SetClusterAuditConfig(ctx, auditConfig); err != nil {
+		return nil, trail.ToGRPC(err)
```

Comment:
This will be an uncaught merge "conflict" once https://github.com/gravitational/teleport/pull/7217 is merged
```suggestion
		return nil, trace.Wrap(err)
	}
	auditConfig, err := auth.ServerWithRoles.GetClusterAuditConfig(ctx)
	if err != nil {
		return nil, trace.Wrap(err)
	}
	auditConfigV2, ok := auditConfig.(*types.ClusterAuditConfigV2)
	if !ok {
		return nil, trace.Wrap(trace.BadParameter("unexpected type %T", auditConfig))
	}
	return auditConfigV2, nil
}

// SetClusterAuditConfig sets cluster audit configuration.
func (g *GRPCServer) SetClusterAuditConfig(ctx context.Context, auditConfig *types.ClusterAuditConfigV2) (*empty.Empty, error) {
	auth, err := g.authenticate(ctx)
	if err != nil {
		return nil, trace.Wrap(err)
	}
	if err = auth.ServerWithRoles.SetClusterAuditConfig(ctx, auditConfig); err != nil {
		return nil, trace.Wrap(err)
```